client.go

```diff
@@ -526,6 +526,13 @@ func (c *Client) ResetBMC(ctx context.Context, resetType string) (ok bool, err e
 	return ok, err
 }
 
+// TerminateSOL pass through library function to terminate active SOL sessions
+func (c *Client) TerminateSOL(ctx context.Context) (ok bool, err error) {
```

Comment:
Include tracing for the client method
```suggestion
func (c *Client) TerminateSOL(ctx context.Context) (ok bool, err error) {
    ctx, span := c.traceprovider.Tracer(pkgName).Start(ctx, "TerminateSOL")
	defer span.End()
```

Reply:
Ack, will do -- thanks.